Problem3.java

```diff
@@ -0,0 +1,33 @@
+// Time Complexity : O(n)
+// Space Complexity : O(n)
```

Comment:
O(52) or O(1), the input will only have Upper and lower case letters.

Reply:
Can you please explain? I mean is it constant?